setup.cfg

```diff
@@ -79,9 +79,10 @@ openstack.metric.v1 =
 [extras]
 test =
   coverage>=3.6
-  python-subunit>=0.0.18
-  testrepository>=0.0.18
   testtools>=1.4.0
+  pytest
+  pytest-cov
```

Comment:
Maybe only add in the cover target? That's the only place someone would need it.

Reply:
done